website/src/layouts/base/header/Navigation.tsx

```diff
@@ -34,50 +35,64 @@ export function Navigation() {
+                        <MegaMenu>
+                            {pathogenMegaMenuSections.map((section) => (
+                                <MegaMenuSection key={section.headline} {...section}>
+                                    {section.navigationEntries.map((entry) => (
+                                        <MegaMenuListEntry
+                                            key={entry.label}
+                                            label={entry.label}
+                                            externalLink={entry.externalLink}
+                                            href={entry.href}
+                                            className={entry.underlineColor}
+                                        />
+                                    ))}
+                                </MegaMenuSection>
+                            ))}
+                        </MegaMenu>
+                    </MegaMenuNavigationEntry>
                 </li>
                 <li className='h-full'>
-                    <NavigationEntry href={Page.dataSources} label='Data sources' />
+                    <SimpleNavigationEntry href={Page.dataSources}>Data sources</SimpleNavigationEntry>
                 </li>
             </ul>
         </nav>
     );
 }
 
-export type NavigationEntryProps = {
-    label: string;
-    href?: string;
+const commonNavigationEntryCss = `flex ${headerHeight} items-center text-gray-500 transition-all hover:border-b-2 hover:border-black hover:text-black`;
+
+export type SimpleNavigationEntryProps = {
+    href: string;
 };
 
-export function NavigationEntry({ href, label, className }: WithClassName<NavigationEntryProps>) {
-    const content = (
-        <div className={`flex ${headerHeight} items-center text-gray-500 transition-all hover:text-black ${className}`}>
-            {label}
-        </div>
+function SimpleNavigationEntry({ href, children }: PropsWithChildren<SimpleNavigationEntryProps>) {
+    return (
+        <a className={commonNavigationEntryCss} href={href}>
+            {children}
+        </a>
     );
+}
+
+const overlayThatClosesMenuOnClickOutside =
+    'group-open:before:bg-gray-500/40 group-open:before:bg-transparent group-open:before:block group-open:before:absolute group-open:before:top-0 group-open:before:left-0 group-open:before:w-full group-open:before:h-[100vh]';
 
-    if (href !== undefined) {
-        return <a href={href}>{content}</a>;
-    }
+const expandableIndicator = 'after:iconify after:ml-1 after:mdi--chevron-down';
 
-    return <button>{content}</button>;
+const openIndicator = 'group-open:border-b-2 group-open:border-black group-open:text-black';
+
+function MegaMenuNavigationEntry({ label, children }: PropsWithChildren<{ label: string }>) {
+    return (
+        <details className='group'>
+            <summary
+                className={`cursor-pointer ${commonNavigationEntryCss} ${overlayThatClosesMenuOnClickOutside} ${expandableIndicator} ${openIndicator}`}
```

Comment:
Do they know that?

But I think we need the cursor to the menu entry itself, because it looks weird if it still has the normal cursor. We could remove the `cursor-pointer` again for the `before` element though.

Reply:
As discussed: we now have the curser when the user hovers over the entry, but not when hovering over the background.